### PR TITLE
Fix confusing SliderJoint3D brief description

### LINKED ISSUE #25631: SliderJoint, Angular Limit on wrong Axis?
**Godot version:**
3.1 beta3

**OS/device including version:**
Windows 10

**Issue description:**
I would call this not a bug, but a design flaw because this class is not working as documented. (IMHO)
The docs say about the function: "Piston kind of slider between two bodies in 3D."

But actually, the slider slides along the X-Axis ....and rotates around the X-Axis. This would allow twisting the connected piston rod but not the typical "up/down" movement. So the expected behaviour of a piston slider would be to allow the rotation either on the z- (preferred) or y-axis when sliding on the x-axis.

Here's a screen shot of a non-working SliderJoint configuration:
sliderjoint

In the foreground you see a slider joint with a linear limit (two cyan rectangles) and an angular limit (90° segment to the left) set. The indicated limits are exactly performed the way they are shown when the game runs. But as I said, the angular limit makes IMHO no sense for a piston like movement.

**Steps to reproduce:**
Add slider joint to a project, edit the linear and angular limits.

**Minimal reproduction project:**
I stepped upon this problem while creating a 3D Joint sampler/demo. I'll add this here when it is ready. But I'll add a workaround (use Generic6DOFJoint instead) to that project to get it working.

### PR DESCRIPTION
**Pull request by Calinou**
This closes https://github.com/godotengine/godot/issues/25631.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

File: doc/classes/Joint3D.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 		Base class for all 3D joints.
 	</brief_description>
 	<description>
-		Joints are used to bind together two physics bodies. They have a solver priority and can define if the bodies of the two attached nodes should be able to collide with each other.
+		Joints are used to bind together two physics bodies. They have a solver priority and can define if the bodies of the two attached nodes should be able to collide with each other. See also [Generic6DOFJoint3D].
 	</description>
```
Comment (madmiraal) [inline doc/classes/Joint3D.xml:8]: I think you got a bit cut and past happy here.

**Review by madmiraal — COMMENTED** [review]
Is it really necessary to reference the `Generic6DOFJoint3D` from every other joint?

File: doc/classes/Joint3D.xml (hunk 1 above)
Comment (Calinou) [reply doc/classes/Joint3D.xml:8]: I know it wasn't really relevant here, but I added it just in case.

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**